CODEOWNERS

```
@@ -35,3 +35,6 @@ k8s/**/cluster-*/static-overlay/static/kustomization.yaml @hmcts/platform-engine
 k8s/**/cluster-*/static-overlay/static/.flux.yaml @hmcts/platform-engineering
 
 ### Next team
+
+##ccd-demo
+k8s/**/common/ccd/ccd-demo.yaml @hmcts/ccd-demo
```

don't wildcard the beginning, provide the full path,
```suggestion
k8s/demo/common/ccd/*.yaml @hmcts/ccd-demo
```

---

modified as suggested